artifacts/mediaTypes.md

```diff
@@ -0,0 +1,211 @@
+
+By supporting new `manifest.config.mediaTypes`, registry tools MAY understand the types stored. 
+
+A simplified example of a registry listing may look like:
+
+| artifact reference | icon | type |
+|-|-|-|
+| `samples/image/hello-world:1.0` |![](../img/oci-container.png)| container image | 
+| `samples/image/hello-world-doc:1.0`|![](../img/doc.png)| doc |
+| `samples/helm/hello-world:1.0` |![](../img/helm.png)| helm chart | 
+
+## Authoring New Artifact Types
+
+Artifact authors have the freedom to define their artifacts as meets their needs. To conform to the OCI Image spec, and be stored within an OCI distribution based registry, Artifact authors MUST follow the following requirements:
+
+- Artifact authors MUST provide unique `config.mediaTypes` to represent their artifact. 
+- Artifact authors MAY submit a PR for their `config.mediaType` to [./mediaTypeMappings.json]() to make their new artifact type well known.
+- Artifact authors MAY provide new schemas for the config.json object. 
+- Artifacts and tooling that use existing `config.mediaTypes` MUST conform to the artifacts spec.
+- Tools MAY choose to support new `config.mediaTypes`.
+- Tools MUST be able to ignore `config.mediaTypes` they don't support. 
+- Distribution implementations MAY choose to support new `config.mediaTypes`. See the [distribution-spec] for more details on processing of the `config.mediaTypes`
+
+### mediaTypes formatting
+
+New mediaTypes MUST use the following convention:
+
+`application/vnd.`[governingOrg/vendor]`.`[artifactName]`.config.`[version]`+json` 
+
+**Example:** 
+The Helm 3 artifact may be represented as: `application/vnd.cncf.helm/config.v3+json`
+
+## Config Parsing
+
+New artifacts MUST choose from 3 options for how the `config` object is processed and parsed:
+
+- Support the [`application/vnd.oci.image.config.v1+json`](../config.md) schema
+- Define a new schema, which artifact specific tools would need to support
+- Define a `null` schema, where the artifact only uses the `config.mediaType` to identify it's unique artifact type. 
+
+Artifact authors MAY publicize their schema in [./mediaTypeMappings.json]() through a pull request.
+
+Artifact authors understand supporting additional artifact types is optional, and distribution parsing of new config schemas is optional. However, new config schemas may be useful to that artifact tooling experience. 
+
+## Layer mediaTypes
+
+While OCI container images have ordinal layers, supporting overlaying of a unified file system, new artifact authors MAY define how they persist their layers, which may be individual files, or collections of files. The tooling specific to that artifact type owns the processing of the files and how they are extracted on the client. 
+
+The [layer mediaType descriptor in the OCI Image spec](https://github.com/opencontainers/image-spec/blob/master/manifest.md#image-manifest-property-descriptions) identifies the `mediaType` MUST support one of the existing layer formats. 
+
```

Comment:
Question! So I'm allowed to do whatever I like for the config mediaType, but then in the layers list, it MUST be one that has been pre-approved? Or is layer formats referring to the array (and not the objects themselves?)

Reply:
If I'm deploying a registry, am I allowed to create my own mediaType for an artifact? (e.g., `application/vnd.dinosaur.dump.v1` or would I need to use only "one of the existing layer formats?"

Reply:
(that's clarification for what I was asking :point_up: )

Reply:
Open for debate - media type constraints should be left to registry implementors. 
I don't know if the reference implementation should include a default validator with a configurable list of valid types. 
/cc @yuwaMSFT2